webextensions/api/tabs.json

```
@@ -3139,6 +3204,28 @@
                 }
               }
             },
+            "attention": {
+              "__compat": {
+                "support": {
+                  "chrome": {
+                    "version_added": false
+                  },
+                  "edge": "mirror",
+                  "firefox": {
+                    "version_added": "63"
+                  },
+                  "firefox_android": {
+                    "version_added": "63",
+                    "notes": "While present, always returns <code>false</code>."
+                  },
```

```suggestion
                  "firefox_android": {
                    "version_added": false
                  },
```

`tabs.query` throws when `attention` is passed, which means that it is not supported in Firefox for Android.